src/metabase/driver/mongo.clj

```diff
@@ -104,22 +106,28 @@
                                  (find-nested-fields field-value nested-fields)
                                  nested-fields)))))
 
+(s/defn ^:private ^:always-validate most-common-object-type :- Class
```

Comment:
Probably muscle memory here on the `^:always-validate` :smiley:

Reply:
actually that's just code from before yesterday
I'll fix that